src/ethereum_spec_tools/lint/lints/glacier_forks_hygiene.py

```diff
@@ -0,0 +1,183 @@
+"""
+Glacier Fork Hygiene Lint
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Ensures that the glacier forks have changes only in BOMB_DELAY_BLOCKS.
+"""
+import ast
+import sys
+from typing import Dict, List, Sequence
+
+from ethereum_spec_tools.forks import Hardfork
+from ethereum_spec_tools.lint import (
+    Diagnostic,
+    Lint,
+    compare_ast,
+    walk_sources,
+)
+
+
+class GlacierForksHygiene(Lint):
+    """
+    Ensures that the glacier forks have changes only in BOMB_DELAY_BLOCKS.
+    """
+
+    def __init__(self) -> None:
+        self.delay_blocks = {
+            "muir_glacier": 9000000,
+            "arrow_glacier": 10700000,
+            "gray_glacier": 11400000,
+        }
```

Comment:
Is it useful to compare a constant in file A with a constant in file B? If someone is going to mess up the fork number, they'll just add the same incorrect number here and under `src/` :rofl:

Reply:
The only other thing that I could think of was to just ignore any comparisons if the node is `BOMB_DELAY_BLOCKS` but then concluded that some comparison is better than nothing :D